pyproject.toml

```diff
@@ -200,7 +200,7 @@ exclude = [
     "build",
     "tests",
     "results",
-    "docs/sphinx-docs-starter-pack",
+    "docs",
```

Comment:
Not checking docs?

Reply:
Just for typing. I don't want to have to include strict typing in every piece of sample code.